templates/site/index.html

```diff
@@ -0,0 +1,209 @@
+{% extends "layouts/main.html" %}
+
+{% block content %}
+
+    {% with page = "home"  %}
+        {% set banner_background_class= "banner-image-home" %}
+        {% set banner_large_background_class = "banner-image-home-large" %}
+        {% set banner_title_class = "banner-title-home" %}
+        {% set banner_video = "video/banner-video.mp4" %}
+        {% set page_title = "Data Analytics for Canadian Climate Services, STAC API" %}
+        {% set page_subtitle = "Climate Data Search" %}
+        {% set page_slogan = "Explore this rich repository of information about Earth’s climate—uncovering insights into its past, understanding current trends, and anticipating future changes." %}
+        {% include "partials/banner.html" %}
+    {% endwith %}
+
+<!--Page content area-->
+<div class="row">
+    <div class="container-fluid content-search-main">
+        <div class="content-search-left">
+        <div id="backButtonContainer" class="div-back-button-container">
+            <i class="fa-solid fa-arrow-left back-button-arrow"></i><a id="backButton" class="button-back-button subtitle-1" role="button">Home</a><img  src="images/open-in-app.svg">
```

Comment:
This button shows up on the search page when the page first loads. I don't know what Home means in this context and if you click it, it takes you to an empty dataset page with no info and a partial map